### DISCUSSION TITLE
How to install dependencies

### DISCUSSION THREAD

**Discussion by Harit007x** [vercel/turborepo#7344] [Help] [answered]
I've just started with the turbo repo and have been trying setup an existing Github repo for practice where they are using the same.
The codebase doesn't have a proper readme, and is already a lot confusing to go through.

If someone can explain the usual steps to do the setup, then I'd be great, thanks.

**Comment by shinokada** [comment, marked as answer]
This doc was helpful for me to get started, https://turbo.build/repo/docs/getting-started/create-new
If you want to install dependencies, you just need to run `pnpm i` after installing turborepo.

**Reply by Harit007x** [reply]
Thanks, following the DOCs was indeed helpful.
